Parsers/ASimWebSession/Test/Cisco_Firepower_ASimWebSession_SchemaTest.csv

```diff
@@ -0,0 +1,114 @@
+"(2) Info: Missing optional field [HttpResponseBodyBytes]"
+"(2) Info: Missing optional field [HttpResponseCacheControl]"
+"(2) Info: Missing optional field [HttpResponseExpires]"
+"(2) Info: Missing optional field [HttpResponseHeaderCount]"
+"(2) Info: Missing optional field [HttpResponseTime]"
+"(2) Info: Missing optional field [HttpVersion]"
+"(2) Info: Missing optional field [NetworkBytes]"
+"(2) Info: Missing optional field [NetworkConnectionHistory]"
+"(2) Info: Missing optional field [NetworkDuration]"
+"(2) Info: Missing optional field [NetworkIcmpCode]"
+"(2) Info: Missing optional field [NetworkIcmpType]"
+"(2) Info: Missing optional field [NetworkPackets]"
+"(2) Info: Missing optional field [NetworkSessionId]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcAppId]"
+"(2) Info: Missing optional field [SrcAppName]"
+"(2) Info: Missing optional field [SrcAppType]"
+"(2) Info: Missing optional field [SrcBytes]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoCountry]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcInterfaceGuid]"
+"(2) Info: Missing optional field [SrcInterfaceName]"
+"(2) Info: Missing optional field [SrcMacAddr]"
+"(2) Info: Missing optional field [SrcNatIpAddr]"
+"(2) Info: Missing optional field [SrcNatPortNumber]"
+"(2) Info: Missing optional field [SrcOriginalUserType]"
+"(2) Info: Missing optional field [SrcPackets]"
+"(2) Info: Missing optional field [SrcUserId]"
+"(2) Info: Missing optional field [SrcVlanId]"
+"(2) Info: Missing optional field [SrcZone]"
+"(2) Info: Missing optional field [ThreatConfidence]"
+"(2) Info: Missing optional field [ThreatField]"
+"(2) Info: Missing optional field [ThreatFirstReportedTime]"
+"(2) Info: Missing optional field [ThreatId]"
+"(2) Info: Missing optional field [ThreatIpAddr]"
+"(2) Info: Missing optional field [ThreatIsActive]"
+"(2) Info: Missing optional field [ThreatLastReportedTime]"
+"(2) Info: Missing optional field [ThreatOriginalConfidence]"
+"(2) Info: Missing optional field [ThreatOriginalRiskLevel]"
+"(2) Info: Missing optional field [ThreatRiskLevel]"
+"(2) Info: Missing optional field [UrlCategory]"
+"(2) Info: Missing optional field [UrlOriginal]"
+"(2) Info: Missing recommended alias [Hostname] aliasing non-existent column [DstHostname]"
+"(2) Info: Missing recommended alias [HttpStatusCode] aliasing non-existent column [EventResultDetails]"
+"(2) Info: extra unnormalized column [CollectorHostName]"
```

Comment:
@jayeshprajapaticrest @niralishah-crest 
I felt we are missing many websession specific fields. Did we try generating all possible logs from live environment?